Sources/Resources/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+	<key>NSPrivacyAccessedAPITypes</key>
+	<array/>
+	<key>NSPrivacyCollectedDataTypes</key>
+	<array/>
+	<key>NSPrivacyTracking</key>
+	<false/>
+	<key>NSPrivacyTrackingDomains</key>
+	<array/>
```

Comment:
This is an empty privacy manifest, so it's not really declaring anything. You don't need anything other than `NSPrivacyAccessedAPITypes`, so you can remove the others.
Under `NSPrivacyAccessedAPITypes`, you'll need to add the `NSPrivacyAccessedAPIType` and `NSPrivacyAccessedAPITypeReasons` as described [here](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api), based on the usage of each sensitive API.

Reply:
> Since there is no usage of sensitive APIs should that just be an empty array?

Do you mean that none of the [sensitive APIs](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files/describing_use_of_required_reason_api) are used at all? Or they are used but fall within the accepted reasons? If it's the former, yes you wouldn't list anything under `NSPrivacyAccessedAPITypes`, but if it's the latter, you'd still want to specify which ones and their reasons.

> I set them all to empty values in order to be explicit, since I saw a comment saying that at least NSPrivacyTracking needed to be included: https://github.com/CocoaPods/CocoaPods/issues/11990#issuecomment-1855075327

I don't believe that's true. Check the note in [this doc](https://developer.apple.com/documentation/bundleresources/privacy_manifest_files):
`You only need to supply NSPrivacyAccessedAPITypes for apps and third-party SDKs on iOS, iPadOS, tvOS, visionOS, and watchOS.`
So there's no need to include it there, I'd keep the `xcprivacy` file focused on the sensitive APIs.

Reply:
Indeed, no sensitive APIs are used at all.

I feel like that note could be interpreted to mean: "on iOS, etc. you need to supply NSPrivacyAccessedAPITypes; on other OS's you don't need to supply it". Given the ambiguity I'll keep these keys here with falsey values as long as it doesn't hurt.

Reply:
I interpreted it as iOS, etc only have to provide `NSPrivacyAccessedAPITypes` but not the other 3 🤷‍♂️ But you're right, it can have two meanings, so it doesn't hurt to play it safe.